client/js/widgets/widget.js

```diff
@@ -362,7 +363,7 @@ export class Widget extends StateManaged {
   }
   css() {
     let css = this.get('css');
-
+    css = this.cssBorderRadius() + css;
```

Comment:
This is to put the value from `borderRadius` before any game provided styles otherwise a game's border-radius in existing css would be overridden by any default `borderRadius` value. The other values have always been tacked on at the end because we do want to override game styles for the widget properties that have always existed. See note about depreciating use of `border-radius` in `css` property in the description.

Reply:
ok